### PR TITLE
Remove comments on public_key_registry

### PR DESCRIPTION
**Pull request by DOBEN**
## Purpose

addresses #https://github.com/Concordium/concordium-rust-smart-contracts/issues/258

## Changes

- Remove comments on `public_key_registry` and `registration_tab`.
- Explain `check_account_signature` host function.
- Add explanation on how to get the schema template with the command`cargo concordium build --schema-template-out -`.

### CODE REVIEW

**Review by dg-concordium — APPROVED** [review]
I made some changes to language directly in the front end/back end file. Otherwise LGTM.

File: source/mainnet/smart-contracts/tutorials/sponsoredTransactions/sponsoredTransactionsFrontendAndBackend.rst [hunk 1]
```diff
@@ -1,77 +1,72 @@
 .. include:: ../../../variables.rst
 .. _sponsoredTransactionsFrontendAndBackend:
 
-===========================================
-Sponsored Transactions Frontend and Backend
-===========================================
+=============================================
+Sponsored Transactions Front end and Back end
```
Comment (abizjak) [inline source/mainnet/smart-contracts/tutorials/sponsoredTransactions/sponsoredTransactionsFrontendAndBackend.rst:5]:
```suggestion
Sponsored Transactions Front End and Back End
```

??

**Review by abizjak — APPROVED** [review]
Looks good.

Can we just deploy the new frontend and then merge & deploy this?

**Comment by DOBEN** [comment]
@abizjak The new front end 2.0.2 is already deployed:
https://sponsored.testnet.concordium.com/

**Comment by DOBEN** [comment]
The new front end 2.0.2 is already deployed:
 https://sponsored.testnet.concordium.com/

**Pull request merged**